paper_figures/code/maxsavings_boxplot.py

```diff
@@ -61,15 +62,30 @@
             baseload=np.ones_like(dam),
         )
 
-# sort the regions by the max savings in either LMP or MEF
-max_savings = np.maximum(mef_savings_sweep.max(axis=1), dam_savings_sweep.max(axis=1), 
-                         aef_savings_sweep.max(axis=1))
+        # Tariff
+        tariff = ps.gettariff(region=reg)
+        startdate_dt, enddate_dt = ms.get_start_end(month)
+        month_length = int((enddate_dt - startdate_dt) / np.timedelta64(1, "h"))
+        tariff_savings_sweep[i, j] = ms.max_tar_savings(
+            data=tariff,
+            system_uptime=0.0,
+            continuous_flex=0.0,
+            baseload=np.ones(month_length),
+            startdate_dt=startdate_dt,
+            enddate_dt=enddate_dt
```

Comment:
Following prev. comment in `maxsavings.py` - tariff optimization for the box plot should optimize uptime.